### PR TITLE
[grafana] Add affinity/node selector / tolerations docs for renderer

### PR DESCRIPTION
**Pull request by thobianchi**
Adds affinity/nodeselector/tolerations in README and in values ( empty ) for image renderer.  
The deployment already support this options.

### CODE REVIEW

**Comment by zanhsieh** [comment]
@thobianchi 
Could you bump the chart version to latest in Chart.yaml? Thank you.

**Comment by zanhsieh** [comment]
@thobianchi 
Could you bump the chart version to latest in Chart.yaml? Thank you.

**Comment by thobianchi** [comment]
done

**Review by zanhsieh — APPROVED** [review]

File: charts/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.26.3
+version: 6.26.4
```
Comment (zanhsieh) [inline charts/grafana/Chart.yaml:3]: 6.26.5 now

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/grafana/Chart.yaml (hunk 1 above)
Comment (thobianchi) [reply charts/grafana/Chart.yaml]: ok done

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**